### PR TITLE
update

### PR DESCRIPTION
**Pull request by jackykinho**
add string

### CODE REVIEW

File: examples/02_control_flow/03_Ranges.md [hunk 1]
```diff
@@ -57,12 +57,12 @@ fun main() {
...
     if (x in 1..10) {            // 1
-        print(x)
+        println("if x is in 1 - 10 range " + "X is "+x)
     }
-    print(" ")
-
-    if (x !in 1..4) {            // 2
-        print(x)
+   
+     val y = 5
```
Comment (p7nov) [inline examples/02_control_flow/03_Ranges.md]: Please don't complicate the example, let's use `x` in both `if`s.

File: examples/02_control_flow/03_Ranges.md [hunk 2]
```diff
@@ -57,12 +57,12 @@ fun main() {
 //sampleStart
     val x = 2
     if (x in 1..10) {            // 1
-        print(x)
+        println("if x is in 1 - 10 range " + "X is "+x)
```
Comment (p7nov) [inline examples/02_control_flow/03_Ranges.md]:
```suggestion
        println("$x is in range from 1 to 10")
```

File: examples/02_control_flow/03_Ranges.md [hunk 3]
```diff
@@ -57,12 +57,12 @@ fun main() {
...
     }
-    print(" ")
-
-    if (x !in 1..4) {            // 2
-        print(x)
+   
+     val y = 5
+    if (y !in 1..4) {            // 2
+        println("if y is not in 1- 4 range " + "Y is "+y)
```
Comment (p7nov) [inline examples/02_control_flow/03_Ranges.md]:
```suggestion
        println("$x is not in range from 5 to 10")
```

Comment (p7nov) [reply examples/02_control_flow/03_Ranges.md]: Please fix the condition of `if` in accordance with the message you print and remove the declaration of `y`.

Comment (jackykinho) [reply examples/02_control_flow/03_Ranges.md]: println("$x is not in range from 5 to 10")

File: examples/02_control_flow/03_Ranges.md (hunk 1 above)
Comment (p7nov) [reply examples/02_control_flow/03_Ranges.md]: Please remove the variable `y` and update the condition in `if`  below

File: examples/02_control_flow/03_Ranges.md [hunk 4]
```diff
@@ -57,12 +57,12 @@ fun main() {
...
-        print(x)
+        println("$x is in range from 1 to 10")
     }
-    print(" ")
-
-    if (x !in 1..4) {            // 2
-        print(x)
+   
+     val y = 5
+    if (y !in 1..4) {            // 2
```
Comment (p7nov) [inline examples/02_control_flow/03_Ranges.md]:
```suggestion
    if (x !in 5..10) {            // 2
```

**Comment by p7nov** [comment]
Sorry, no answer so I'm closing this.

**Pull request closed without merging**